integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -39,7 +38,7 @@
         var d = document;
         var pbs = d.createElement("script");
         pbs.type = "text/javascript";
-        pbs.src = '/build/dist/prebid.js';
+        pbs.src = '/build/dev/prebid.js';

```

Comment:
If they copied this verbatim, then it probably wouldn't work unless they host the script on their site at `/build/dist/prebid.js`.  So it should be pretty obvious they would need to set the path correctly. I'd suggest a comment rather than make the integrating testing more difficult.